.palantir/revapi.yml

```diff
@@ -1285,6 +1285,12 @@ acceptedBreaks:
     - code: "java.field.removedWithConstant"
       old: "field org.apache.iceberg.TableProperties.ROW_LINEAGE"
       justification: "Removing deprecations for 1.10.0"
+    - code: "java.method.abstractMethodAdded"
+      new: "method <T extends org.apache.iceberg.rest.RESTResponse> T org.apache.iceberg.rest.BaseHTTPClient::execute(org.apache.iceberg.rest.HTTPRequest,\
+        \ java.lang.Class<T>, java.util.function.Consumer<org.apache.iceberg.rest.responses.ErrorResponse>,\
+        \ java.util.function.Consumer<java.util.Map<java.lang.String, java.lang.String>>,\
+        \ org.apache.iceberg.rest.ParserContext)"
+      justification: "Add context aware parsing"
```

Comment:
nit: The Justification here is that no one can depend on this api correct? It's solely owned by the client and no one else can extend or use it?